templates/answerfile_3.6_basic.txt.j2

```diff
@@ -0,0 +1,4 @@
+{% include "./templates/basic_answerfile.txt.j2" %},
+OVESETUP_DB/upgradeWithHeEl6Hosts=none:None
```

Comment:
You do not really need these. Generally speaking, almost all vars default to None, so setting them to None in the answerfile almost always does nothing. But no harm in keeping them anyway.

Reply:
I removed this file.